ocrd/ocrd/cli/workspace.py

```diff
@@ -598,7 +598,7 @@ def merge(ctx, copy_files, filegrp_mapping, file_grp, file_id, page_id, mimetype
         fileGrp_mapping=filegrp_mapping,
         fileGrp=file_grp,
```

Comment:
Should also apply to `fileGrp` vs `file_grp`:
```suggestion
        file_grp=file_grp,
```

Reply:
fixed in 35f854cda